src/NuGet.Clients/NuGet.CommandLine/Commands/SearchCommand.cs

```diff
@@ -0,0 +1,208 @@
+                {
+                    Console.WriteLine(new string('-', 20));
+                    Console.WriteLine("No results found.");
+                    Console.WriteLine(new string('-', 20) + "\n");
+                }
+                else
+                {
+                    if (Verbosity == Verbosity.Quiet)
+                    {
+                        System.Console.WriteLine($"Source: {source.Name}");
+                        System.Console.WriteLine(new string('-', 20));
+                    }
+                    PrintResults(results);
+                }
+            }
+        }
+
+        private void PrintResults(IEnumerable<IPackageSearchMetadata> results)
+        {
+            foreach (IPackageSearchMetadata result in results)
+            {
+                Console.WriteLine(new string('-', 20));
+
+                CultureInfo culture = CultureInfo.CurrentCulture;
+
+                string printBasicInfo = $"> {result.Identity.Id} | {result.Identity.Version.ToNormalizedString()}";
+
+                string downloads, printDownloads;
+
+                if (result.DownloadCount != null)
+                {
+                    downloads = string.Format(culture, "{0:N}", result.DownloadCount);
+                    printDownloads = $" | Downloads: {downloads.Substring(0, downloads.Length - 3)}";
+                }
+                else
+                {
+                    printDownloads = " | Downloads: N/A";
+                }
+
+                System.Console.WriteLine(Verbosity != Verbosity.Quiet ? printBasicInfo + printDownloads : printBasicInfo); // System.Console is used so that output is not suppressed by Verbosity.Quiet
+
+                if (Verbosity != Verbosity.Quiet && result.Description != null)
+                {
+                    string description = result.Description;
+
+                    if (Verbosity == Verbosity.Normal)
+                    {
+                        if (description.Length > 100)
+                        {
+                            description = description.Substring(0, 100) + "...";
+                        }
+                    }
+                    Console.PrintJustified(2, description);
+                }
+            }
+
+            Console.WriteLine(new string('-', 20));
+            System.Console.WriteLine("");
+        }
+
+        public override bool IncludedInHelp(string optionName)
+        {
+            if (string.Equals(optionName, "ConfigFile", StringComparison.OrdinalIgnoreCase))
+            {
+                return false;
+            }
+
+            return base.IncludedInHelp(optionName);
+        }
+        
+        private static string RemoveSchemaNamespace(string content)
+        {
+            // This seems to be the only way to clear out xml namespaces.
+            return Regex.Replace(content, @"(xmlns:?[^=]*=[""][^""]*[""])", string.Empty, RegexOptions.IgnoreCase | RegexOptions.Multiline | RegexOptions.CultureInvariant);
```

Comment:
This is validating that the value=value and quotes match up. Do you really care about that? Seems you could just replace lines starting with "(xmlns" and ending with ")" ?

Reply:
This was dead code, and has been removed in later commits.